learnable/src/pages/NewWorkshop.jsx

```diff
@@ -1,9 +1,16 @@
 import React from 'react'
 import CreateWorkshopForm from '../components/CreateWorkshopForm/CreateWorkshopForm';
 
-function NewWorkshopPage() {
-    return <CreateWorkshopForm />
+// function NewWorkshopPage() {
+//     return <CreateWorkshopForm />
+
+// };
 
+function NewWorkshopPage() {
+    const token = window.localStorage.getItem("token")
+    // console.log( token !== null)
```

Comment:
```suggestion
```

Reply:
If you hit the 'commit suggestion' button here, the code edit I've suggested will be added to your branch. Note: You will have to run "git pull" in your branch from your terminal to pull the changes to your local copy of the code on your machine. :-) Otherwise you can go in on your machine and make the changes yourself and 'git push' to here instead. Whatever feels easier for you